Dockerfile.apk

```diff
@@ -1,9 +1,9 @@
-FROM alpine
-
-RUN apk update \
-  && apk add ca-certificates wget coreutils patch grep \
-  && update-ca-certificates \
-  && apk add --virtual build-deps make gcc make g++ zlib-dev musl-dev pcre-dev \
-    openssl-dev zlib-dev ncurses-dev readline-dev curl perl \
-    libffi-dev git unzip tar openssh m4 xz gnupg openssl musl \
-    linux-headers bsd-compat-headers bash
+FROM alpine
+
+RUN apk update \
+  && apk add ca-certificates wget coreutils patch grep \
+  && update-ca-certificates \
+  && apk add --virtual build-deps make gcc make g++ zlib-dev musl-dev pcre-dev \
+    openssl-dev zlib-dev ncurses-dev readline-dev curl perl \
+    libffi-dev git unzip tar openssh m4 xz gnupg openssl musl \
+    linux-headers bsd-compat-headers bash go
```

Comment:
Is the `go` package required here (if we are downloading the binaries)?

Reply:
It is as the installation of go via the script I stole from the docker golang doesn't apply to alpine